cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -67,7 +80,12 @@ def _cleanup():
         migration_plan.name, view.migration_plans_completed_list.get_vm_count_in_plan(
             migration_plan.name), view.migration_plans_completed_list.get_clock(
             migration_plan.name))
+    # validate MAC address matches between source and target VMs
     assert view.migration_plans_completed_list.is_plan_succeeded(migration_plan.name)
+    src_vm = form_data_vm_obj_single_datastore.vm_obj[0]
+    migrated_vm = get_migrated_vm_obj(src_vm, v2v_providers.rhv_provider)
+    assert src_vm.mac_address == migrated_vm.mac_address
+    assert src_vm_ip_addr == migrated_vm.ip_address
```

Comment:
I'm not sure this is necessarily something that we can validate against, as its highly dependent on DHCP server and client configuration. The target provider may be served by a different DHCP server even, in a different subnet, than the source machine. I think it should only validate that the MAC addresses match, as that's all the migration guarantees.

Reply:
Should we maybe make it soft_assert?

Reply:
@jawatts it is a test case to see 1) if the MAC and 2) IP Addresses remain same, but IP address is best attempt, not a guarantee.